src/Migrations/20240123130535_RemoveJobIdColumn.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace NodeGuard.Migrations
+{
+    /// <inheritdoc />
+    public partial class RemoveJobIdColumn : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "JobId",
+                table: "WalletWithdrawalRequests");
+
+            migrationBuilder.DropColumn(
+                name: "JobId",
+                table: "Nodes");
+
+            migrationBuilder.DropColumn(
+                name: "JobId",
+                table: "ChannelOperationRequests");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Discriminator",
+                table: "AspNetUsers",
+                type: "character varying(21)",
+                maxLength: 21,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "text");
```

Comment:
@Jossec101 I can't find anything about this change in Microsoft documentation or npsql issues. Should we keep it?

Reply:
Looks like a upgrade on NGPSQL did this :/ Lets keep it